GradDemo.Api/GradDemo.Api.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Coinbase" Version="6.0.1" />
```

Comment:
Why did you add Coinbase?


Also loads and loads of files added for a simple api call